bin/cli

```diff
@@ -24,8 +24,9 @@ const gitignore = `
 node_modules/
 
 # Prototype ignores - per-user
-.env
 public/
+.env
+usage-data-config.json
```

Comment:
Okay, I'll do that instead.

Reply:
Hmm, that would mean though that if a user needs to clear their tmp folder because of an issue, they would lose this setting. Is that okay?

Reply:
I think that's a better compromise than the user having that file in their working directory - I'm keen to keep the working directory as clean as possible.  I think moving it is the right thing because:

  - I don't expect the average user to need to clear their `.tmp` directory regularly
  - If they do clear it I think it's reasonable to behave the same as a fresh checkout (and ask the question)
  - It keeps the working directory clearer which I think will really help our users
  - It's easy(ish) to revert in a later version if that was the wrong approach

I've just checked our clean to make sure this wouldn't be wiped out and ... I'll raise a ticket to update our clean task :) https://github.com/alphagov/govuk-prototype-kit/blob/v13/lib/build/tasks.js#L60